### ISSUE TITLE
support documentation in regex datatype

### ISSUE THREAD

**Issue by josteinaj** [daisy/pipeline-webui#77]
``` xml
<data type="string">
    <param name="pattern">(\([^():]+:[^():]+\))*</param>
    <documentation>The query must be a list of parameters, where a parameter starts with a left paranthesis, then the name of the parameter, then a colon, then the value of the parameter, and then finally a right paranthesis. Spaces are not allowed before or after parameters.</documentation>
</data>
```

The text would be displayed when a bad value is entered in the string widget.

**Comment by bertfrees** [comment]
@josteinaj Close?

**Comment by josteinaj** [comment]
Yes, seems to have been fixed in #19.
